src/Facebook/InstantArticles/Transformer/Getters/DateGetter.php

```diff
@@ -50,10 +50,10 @@ public function get($node)
         if (!empty($elements) && $elements->item(0)) {
             $element = $elements->item(0);
 
-            if ($this->attribute) {
+            if ($this->attribute && $this->attribute != 'dateTextContent') {
```

Comment:
Let's not use a custom attribute name like 'dateTextContent'. We can simply omit the attribute on the rule to fetch the content, that's the default for all rules and getters.